app/assets/javascripts/provider/admin/apiconfig/services/service_discovery.js.coffee

```diff
@@ -0,0 +1,59 @@
+$ ->
+  form_source = $('form#new_service_source')[0]
```

Comment:
I'd stay with jQuery

Reply:
If you do not want jQuery then `getElementById`

Reply:
`getElementById` might be better and we'd get rid of jquery

Reply:
well, we are using jquery extensively anyways